cmd/user/internal/application/oauth2/config.go

```diff
@@ -8,15 +8,22 @@ import (
 	"github.com/vardius/go-api-boilerplate/cmd/user/internal/application/config"
 )
 
-// NewConfig provides oauth2 config
-func NewConfig() oauth2.Config {
```

Comment:
This method was providing our internal oauth2 config

Reply:
Yes if the token gets generate in the user service without the need of the oauth2 server becomes obsolete.